hw_7.py

```diff
@@ -0,0 +1,11 @@
+"""
+Write a program to count the number of lines in a text file.
+"""
+
+
+def count_rows():
+    with open('documentation file/short documentation.txt', 'r') as f:
+        return sum(1 for x in f)
```

Comment:
It's not clear what happened here.

Reply:
the loop goes through each line and writes each line as "1" to the list, after which everything is added

Reply:
It's not clear from this implementation.
It might be done easier.

Reply:
done